### ISSUE TITLE
Unrecoverable crash

### ISSUE THREAD

**Issue by LoBlanc** [StrangeLoopGames/EcoIssues#15832]
Hi, I was going to perform labor on a cement kiln for a work party.
My game crashed and after launching again, I can't get into the game scene.
After the load process, the game returns me to the main menu.
I'm attaching Player.log: [https://drive.google.com/file/d/1tHEOX7CssTFNrrsfMO8RXr6v6wpxP268/view?usp=sharing](url)

**Comment by LoBlanc** [comment]
After trying to login one more time, I've been able to see there was an ingame crash window report, but it faded out too fast to be able to read or take a screen capture

**Comment by ThePiachu** [comment]
It's something to do with my cement contract job on the test server. It crashed both Hobbit and now rev.

1) Version
0.9 Staging-1514

2) Steps to reproduce
- Build a cement kiln, start a job, put the job on the contract board
- Have the output pipe blocked by items in a stockpile?
- Someone tries doing the job, they crash out of the game?
- They can't log back in?

3) Expected behaviour
- Labour should be applied normally

4) Actual behaviour
- People crash out of the game hard and can't log back in

**Comment by revtje** [comment]
Here are my logs. I accepted the contract and went into the room. I did not click on any labor or touch the cement kiln

Version: 0.9.0.0 beta staging-1514
Steps to reproduce:
1. Accept ThePiachu:s work party
2. Go into his industry room
Expected Behavior: No crash and able to start labor his cement kiln.
Actual Behavior: Unable to play due to crash. It crashes the second I am logged back in

[PlayerLogs.zip](https://github.com/StrangeLoopGames/EcoIssues/files/4491741/PlayerLogs.zip)
